src/views/Login.vue

```diff
@@ -61,11 +64,26 @@ export default {
     }
   },
   methods: {
-    onSubmit() {
-      console.log("You're in!");
+    generateLoginObject () {
+      return {
+        email: this.email,
+        password: this.password
+      }
+    },
+    async onSubmit () {
+      if (!this.$v.$invalid) {
+        const userData = this.generateLoginObject()
+        const response = await login(userData)
+        if (response.ok) {
+          console.log(response.data.access_token)
```

Comment:
you have to set the token in the local storage here, are you doing it in another pr?

Reply:
This PR correspond to the Login card in which you just have to print the token. In the next PR, that correspond to the card Session management & Logout I'll be doing what you say.